rust/processor/src/processors/default_processor.rs

```diff
@@ -23,20 +23,34 @@ use diesel::{
     query_builder::QueryFragment,
     ExpressionMethods,
 };
+use serde::{Deserialize, Serialize};
 use std::fmt::Debug;
-use tokio::join;
 use tracing::error;
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct DefaultProcessorConfig {
+    pub skip_move_resources: Option<bool>,
+    pub skip_transactions: Option<bool>,
+    pub skip_write_set_changes: Option<bool>,
```

Comment:
It might be better to make a vec of strings (table names) b/c this could get tedious as we remove more tables.